configs/katello/4.12.yaml

```diff
@@ -0,0 +1,116 @@
+---
+:project: katello
+:github_org: katello
+:nightly: false
+
+:releases:
+  :4.12.0:
+    :redmine_version_id: 1760
+:prior-releases:
+  :4.11.0:
+    :redmine_version_id: 1732
+  :4.10.1:
+    :redmine_version_id: 1774
+
+:code_name: Mile 42
+:mash_scripts:
+  - katello-4.12-el8
+  - katello-candlepin-4.12-el8
+
+:ignores:
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-4.12
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/3.10
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 1.12.z
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 4.3.0
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.9.0
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
+    :branch: 0.5.20
+    :repo: https://github.com/theforeman/foreman_virt_who_configure.git
+  :smart_proxy_container_gateway:
+    :branch: 1.2.0
+    :repo: https://github.com/Katello/smart_proxy_container_gateway.git
+  :katello-selinux:
+    :branch: 5.0.2
+    :repo: https://github.com/Katello/katello-selinux.git
+
+:tags:
```

Comment:
We're using COPR instead of Koji (since Katello 4.11) so this whole section is no longer used. I don't know if tool_belt can deal with this, but if it can't, the code should be modified to do so.

Reply:
Ok, thank you for the review Ewoud. Everything in the tags section can be left out? Would it make sense to remove the tags section from the nightly file as well, in that case?

Reply:
Yes, it would. It may be that some tools within tool_belt still demand it somehow. If so, those tools need to be updated. With https://github.com/theforeman/tool_belt/pull/574 I've made a start.